### PR TITLE
Add Cli Command documentation

### PR DESCRIPTION
**Pull request by AdamJNavarro**
# Why

Provides an added resource for users to gain knowledge of the commands offered by the CLI and what their options do.

# How

Each command is now documented to indicate if an alias is available, the possible options as well as descriptions for the command itself and its options.

Will add changes to other versions upon feedback and approval.

### CODE REVIEW

**Comment by brentvatne** [comment]
Screen Shot 2019-09-26 at 15 50 56

we should improve spacing here - there should be less spacing between the header and its related paragraph and more between the header and the previous section.

<hr />

another concern i have is that this is a lot of information and it's hard to parse. i think we might want to make the commands appear at the top level but have all of the information under the commands collapsed, so you have to click on the command to see the full info. probably want to add an arrow next to it to indicate its expandable. if we don't have any tools for building this into the docs right now we should create them.

<hr />

not sure what the process for creating these docs is like but once we settle on the exact format (the above two points) we should create a script that will generate these docs from `expo-cli` automatically so we don't have to manually update all the time

**Review by brentvatne — CHANGES_REQUESTED** [review]
☝️

File: docs/pages/versions/unversioned/workflow/expo-cli.md [hunk 1]
```diff
@@ -18,68 +18,508 @@ You may use the CLI in your terminal or use the web based interface (it opens au
...
+| Option         | Description             |
+| ------------ | ----------------------- |
+| `--event` [type] | The type of webhook: [build]. |
+| `--config` [path] | Specify a path to app.json. |
+
+**whoami**
+
+Checks with the server to see if you are logged in and if you are, returns what Expo account you are logged in as.
+
+## Universal Options
```
Comment (sjchmiela) [inline docs/pages/versions/unversioned/workflow/expo-cli.md]: I think this should come above the specific commands, otherwise it gets lost

**Review by sjchmiela — APPROVED** [review]
LGTM apart from that one small comment. BTW is this docs easily updatable? (Does it require manual labor to keep it up to date?)

**Comment by sjchmiela** [comment]
Oh, I see this should have gotten closed by https://github.com/expo/expo/commit/1247cefb5b8a2ce32d5f70daf164c7a78506f958.
